src/KM_Utils.pas

```diff
@@ -199,21 +200,28 @@ function UTCToLocal(Input: TDateTime): TDateTime;
 {$ENDIF}
 
 
-function MapSizeText(X,Y: Word): UnicodeString;
+function MapSizeIndex(X, Y: Word): Byte;
 begin
   //Pretend these are understandable in any language
```

Comment:
This comment is referring to the strings XS, S, M, etc. so should be moved there